### PR TITLE
Feat: added featured product section

### PR DESCRIPTION
**Pull request by abhijeetekad**
chrome-capture (5)

### CODE REVIEW

**Review by deepak29-git — APPROVED** [review]

**Review by pratik-kumar-sahu — APPROVED** [review]
- Use `prettier` to format the code.
- get rid of comments in final merge.
- UI looks nice go ahead with merging

**Review by Hardiegogo — COMMENTED** [review]
UI looks really good.
Just a small change suggested below.

File: css/index.css [hunk 1]
```diff
@@ -213,6 +213,120 @@
...
+  height: 20rem;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.brand img {
+  width: 100%;
+}
+.brand p {
```
Comment (Hardiegogo) [inline css/index.css]: Try using classes instead of the descendant selector.

**Review by Hardiegogo — COMMENTED** [review]
UI looks really good.
I have suggested a small change below.

File: css/index.css (hunk 1 above)
Comment (abhijeetekad) [reply css/index.css]: sure.. Thanks for the review and suggestion.

Comment (abhijeetekad) [reply css/index.css]: Done

**Pull request merged**